skore-ui/src/stores/project.ts

```diff
@@ -318,6 +302,42 @@ export const useProjectStore = defineStore("project", () => {
     currentViewItems.value = r;
   }
 
+  /**
+   * Send the view's layout to backend
+   * @param view the view to persist
+   * @param layout the layout to persist
+   */
+  async function _persistView(view: string, layout: Layout) {
+    if (items.value && views.value) {
+      const r = await putView(view, layout);
+      if (r) {
+        setProject(r);
+      }
+    }
+  }
+
+  /**
+   * Persist the current view's layout when the items change.
+   * Useful when user reoganizes items in the current view.
```

Comment:
```suggestion
   * Useful when user reorganizes items in the current view.
```
I wonder why `typos` didn't catch this :thinking: